utils/codepointtrie/src/codepointtrie.rs

```diff
@@ -0,0 +1,503 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Note: GitHub thinks this file was added instead of moved, probably due to the big blob at the bottom.  Not a problem; I just checked the diff manually using another program.